--- freetype1-1.4pre.cvs20060210/ltconfig~	2014-04-08 20:27:20.949283290 +0200
+++ freetype1-1.4pre.cvs20060210/ltconfig	2014-04-08 20:29:05.293394118 +0200
@@ -2944,7 +2944,7 @@ hardcode_action=$hardcode_action
 
 # Flag to hardcode \$libdir into a binary during linking.
 # This must work even if \$libdir does not exist.
-hardcode_libdir_flag_spec=$hardcode_libdir_flag_spec
+hardcode_libdir_flag_spec=
 
 # Whether we need a single -rpath flag with a separated argument.
 hardcode_libdir_separator=$hardcode_libdir_separator
