--- freetype1-1.4pre.cvs20060210/lib/tttypes.h.typedef~	2014-04-08 18:08:09.597474532 +0200
+++ freetype1-1.4pre.cvs20060210/lib/tttypes.h	2014-04-08 18:08:12.415342995 +0200
@@ -32,6 +32,20 @@
  compilation of this library.  Please remove the precompiled headers."
 #endif
 
+#if SIZEOF_INT == 4
+
+  typedef signed int      TT_Int32;
+  typedef unsigned int    TT_Word32;
+
+#elif SIZEOF_LONG == 4
+
+  typedef signed long     TT_Int32;
+  typedef unsigned long   TT_Word32;
+
+#else
+#error "no 32bit type found"
+#endif
+
   typedef          char   String;
   typedef signed   char   Char;
   typedef unsigned char   Byte;
